src/components/SearchBar/index.jsx

```diff
@@ -94,23 +160,50 @@ export default class SearchBar extends Component {
           placeholder="Search"
           className={`react-lazylog-searchbar-input ${searchInput}`}
           onChange={this.handleSearchChange}
-          onKeyPress={this.handleSearchKeyPress}
+          onKeyPress={this.handleKeyPress}
           value={this.state.keywords}
           disabled={disabled}
+          ref={this.inputRef}
         />
         <button
           disabled={disabled}
           className={`react-lazylog-searchbar-filter ${
             filterActive ? 'active' : 'inactive'
-          } ${button} ${filterIcon}`}
-          onClick={this.handleFilterToggle}>
+          } ${button} ${filterIcon} ${clickable}`}
+          onKeyPress={this.handleKeyPress}
+          onMouseUp={this.handleFilterToggle}>
           <FilterLinesIcon />
         </button>
+        {searchLikeBrowser && (
+          <Fragment>
+            <button
+              disabled={disabled}
+              className={`react-lazylog-searchbar-up-arrow ${
+                resultsCount ? `active ${clickable}` : 'inactive'
+              } ${button} ${arrowIcon}`}
+              onKeyPress={this.handleKeyPress}
+              onMouseUp={onShiftEnter}>
+              <UpArrowIcon />
+            </button>
+            <button
+              disabled={disabled}
+              className={`react-lazylog-searchbar-down-arrow ${
+                resultsCount ? `active ${clickable}` : 'inactive'
+              } ${button} ${arrowIcon}`}
+              onKeyPress={this.handleKeyPress}
+              onMouseUp={onEnter}>
+              <DownArrowIcon />
+            </button>
+          </Fragment>
+        )}
+
         <span
           className={`react-lazylog-searchbar-matches ${
             resultsCount ? 'active' : 'inactive'
           } ${resultsCount ? active : inactive}`}>
-          {resultsCount} {matchesLabel}
+          {searchLikeBrowser && resultsCount
+            ? `${currentResultsPosition + 1} of ${resultsCount} ${matchesLabel}`
```

Comment:
I totally can, but I don't think it looks very good. A bit confusing.
![Screen Shot 2021-10-05 at 11 07 11 AM](https://user-images.githubusercontent.com/43556453/136069927-e8d3f067-da6c-45f5-81f2-4cc00ccf90da.png)

What if I just make it a bit wider to begin with so that it doesn't move the buttons by expanding?

Reply:
yeah, that sounds good.